scripts/install.go

```diff
@@ -0,0 +1,62 @@
+package scripts
+
+import (
+	"embed"
+	"fmt"
+	"io/fs"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+//go:embed setup/setup.sh
+//go:embed setup/config-template.toml
+//go:embed setup/coriolis-snapshot-agent.service.sample
+var setupContent embed.FS
+
+const (
+	setupTmpDir = "/tmp/setup"
+)
+
+func writeEmbeddedFile(filename string, mode fs.FileMode) error {
```

Comment:
This should probably receive a destination folder.

```suggestion
func writeEmbeddedFile(filename string, destination string,  mode fs.FileMode) error {
```